test/src/history/code_history_controller_test.dart

```diff
@@ -76,10 +76,8 @@ void main() {
           final controller = await pumpController(wt, MethodSnippet.full);
           await wt.cursorEnd();
 
-          controller.value = controller.value;
           controller.value = controller.value.typed('a');
           controller.value = controller.value.typed('b');
-          controller.value = controller.value;
```

Comment:
I think it is useful to test that changing to the same value doesn't create a history record